packages/libp2p-inspector-metrics/src/index.ts

```diff
@@ -200,7 +200,7 @@ class InspectorMetrics implements Metrics, Startable {
           this.log.error('error sending metrics', err)
         })
       }
-    })({})
+    })({ logger: components.logger })
```

Comment:
Maybe just pass the whole components map in for flexibility:

```suggestion
    })(components)
```